### PR TITLE
Fix #3884 Build time listed in the footer is not Europe/Amsterdam timezone.

### LINKED ISSUE #3884: Build time listed in the footer is not Europe/Amsterdam timezone.
I just built and deployed the 1.11.0 release candidate.
Job started at 12:59 PM.
I've compared MD5 hashes for the created molgenis-app.war and the ROOT.war I've deployed. They match.

Expected:
In the footer it says [...] 1.11.0-SNAPSHOT built on 2015-10-13 13:00.

Actual:
In the footer it says [...] 1.11.0-SNAPSHOT built on 2015-10-13 11:00.
Probably this is UTC timezone?

### PR DESCRIPTION
**Pull request by Mark-de-Haan**

### CODE REVIEW

File: molgenis-core-ui/src/main/java/org/molgenis/ui/MolgenisMenuController.java [hunk 1]
```diff
@@ -38,16 +42,20 @@
 
 	@Autowired
 	public MolgenisMenuController(MolgenisUi molgenisUi, @Value("${molgenis.version}") String molgenisVersion,
-			@Value("${molgenis.build.date}") String molgenisBuildData)
+			@Value("${molgenis.build.date}") String molgenisBuildData) throws ParseException
```
Comment (fdlk) [inline molgenis-core-ui/src/main/java/org/molgenis/ui/MolgenisMenuController.java]: Surely not?

Comment (Mark-de-Haan) [inline molgenis-core-ui/src/main/java/org/molgenis/ui/MolgenisMenuController.java]: Definitely not! :p

**Comment by fdlk** [comment]
Alternatively:
    DateTimeFormatter.ofLocalizedDateTime(FormatStyle.MEDIUM, FormatStyle.SHORT)
				.format(LocalDateTime.now())

**Comment by fdlk** [comment]
Closing for #4531

**Pull request closed without merging**